samples/Core-Bot/datatypes_timex_expression/time.py

```diff
@@ -0,0 +1,17 @@
+
+class Time:
```

Comment:
Nit: Might want to put copyright headers in all stuff for datatypes_timex_expression